modules/front-end/src/app/core/components/metric-drawer/metric-drawer.component.html

```diff
@@ -51,8 +51,8 @@ <h3 i18n="@@expt.metric.event-info">Event information</h3>
         <nz-form-control i18n-nzErrorTip="@@expt.overview.event-type-cannot-be-empty" nzErrorTip="Event type cannot be empty">
           <nz-select formControlName="eventType" (ngModelChange)="onEventTypeChange($event)">
             <nz-option [nzValue]="customEventType"  nzLabel="Custom"></nz-option>
-            <nz-option [nzValue]="pageViewEventType" nzLabel="PageView"></nz-option>
-            <nz-option [nzValue]="clickEventType" nzLabel="Click"></nz-option>
+            <!-- <nz-option [nzValue]="pageViewEventType" nzLabel="PageView"></nz-option>
+            <nz-option [nzValue]="clickEventType" nzLabel="Click"></nz-option> -->
```

Comment:
why comment

Reply:
We need to leave this part for now. we will uncomment when another part is ready. We now support only custom event for an easy mode.

Reply:
why leave for now?